executable/SummarizeApp.re

```diff
@@ -272,17 +185,8 @@ let notification =
             ~resolve=(info, p) =>
             p.payload
           ),
-          field(
-            "messages",
-            ~typ=messagesConnection,
-            ~args=
-              Arg.[
-                arg("first", ~typ=float),
-                arg("after", ~typ=string),
-                arg("last", ~typ=int),
-                arg("before", ~typ=string),
-              ],
-            ~resolve=(_info, p, first, after, last, before) =>
+          MessageConnection.connectionResolver(
+            "messages", (_info, p, first, after, last, before) =>
```

Comment:
thinking why aren't we calling this `notifications`? 🤔

Reply:
we have notifications and inside of it messages